### PR TITLE
Feature/front end tech stack 2016

### PR DESCRIPTION
**Pull request by dennishn**

### CODE REVIEW

File: source/our-stacks/front-end/index.md [hunk 1]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
 layout: stack
 ---
-Total FE
\ No newline at end of file
+The front-end team at Nodes builds a variety of products using modern web technologies.
+We build web apps, hybrid apps and websites using [Angular](https://www.angular.io) or [Ionic](https://www.ionicframework.com) for both our clients and for internal use, as well as the occasional email layouts. 
+
+### Tasks
+ - Writing and maintaining UI libraries for client and internal use
+ - Building responsive layouts for a of modern browsers
```
Comment (nickskull) [inline source/our-stacks/front-end/index.md]: remove "a of" :)

File: source/our-stacks/front-end/index.md [hunk 2]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
\ No newline at end of file
+The front-end team at Nodes builds a variety of products using modern web technologies.
+We build web apps, hybrid apps and websites using [Angular](https://www.angular.io) or [Ionic](https://www.ionicframework.com) for both our clients and for internal use, as well as the occasional email layouts. 
+
+### Tasks
+ - Writing and maintaining UI libraries for client and internal use
+ - Building responsive layouts for a range of modern browsers
+ - Writing business and view logic
+ - Integrating with in-house or external APIs and services 
+ - Work closely with our UX and Design teams
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: work -> working (to match the tense of the other points)

File: source/our-stacks/front-end/index.md [hunk 3]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
+
+#### HTML/Templates
+HTML is a given but we also use the blade templating language when working on PHP projects, and EJS/Handlebars for internal sites and email layouts.
+
+### Frameworks of choice
+
+#### Angular
+[Angular](https://www.angularjs.org) is an open-source MV* framework for building single-page applications (SPAs). It has great features like Dependency Injection(DI), Two-way Bindings (model<-->view), Routing, Animation support built-in, Superb control over forms and much much more.
+  
+We have been using Angular since version 1.1 for production and have released a slew of SPAs with this framework over the years. We feel that this framework has a enterprise level quality, and will continue to build our products with the latest 2.0 version.
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: a enterprise-> an enterprise (or just enterprise by itself)

File: source/our-stacks/front-end/index.md [hunk 4]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
+
+#### Ionic
+The [Ionic](https://www.ionicframework.com) framework builds on top of Angular and gives us a set of tools to build both Native Apps as well as progressive web apps (PWAs). It offers a beautiful UI and access to native features like Cameras, File systems and more through Cordova.
+  
+Being experienced with Angular, this framework was a natural choice for us as we were already familiar with the structures and syntax of Angular which is heavily utilized in Ionic. Through their CLI tools we can easily build apps ready to be deployed to the app stores as well as offering companion web apps sharing most of the same code base.
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: offering -> offer

File: source/our-stacks/front-end/index.md [hunk 5]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
+
+### IDE
+Our IDE of choice is WebStorm/PHPStorm. Before we made the switch to actual IDEs (and boy are we glad we did), we used Sublime Text 2.
+
+### Other tools
+ - [Yeoman](http://yeoman.io/)* - He calls himself your friendly buttler, Yeoman is a CLI scaffolding tool that is easy to extend. We use this every day to scaffold out new parts of our applications using well defined templates.
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: buttler -> butler

File: source/our-stacks/front-end/index.md [hunk 6]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
+ - [Gulp](http://gulpjs.com/)* - A task runner, we are currently working on moving all of our tasks from Grunt to Gulp because of its extensibility and tasks in parallel features.
+ - [Grunt](http://gruntjs.com/)* - A task runner, Grunt in our case is currently used with our boilerplate for client products. 
+ - [Webpack](https://webpack.github.io/) - An amazing module bundler.
+ - [NPM](https://www.npmjs.com/)* - The defacto JavaScript package manager.
+ - [Bower](http://bower.io/)* - A package manager designed for consuming client-side modules and plugins.
+ - [Hexo](http://hexo.io/) - A blogging system build with NodeJS, it uses a flat-file structure much like Jekyll. 
+ - [Postman](https://www.getpostman.com/) - All our internal APIs are documented by our back-end developers through Postman, an app that let's us easily discover the endpoints we need to integrate against.
+ - [Zeplin](https://zeplin.io/) - Zeplin bridges the gap between developers and designers, it let's us easily extract the graphical assets as well as colours and measurements from the design deliverables.
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: colours -> colors (just kidding 😄 )

File: source/our-stacks/front-end/index.md [hunk 7]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
+ 
+### Libraries
+These are the 3rd party libraries we primarily use in our projects.
+
+ - [Angular.js](https://www.angularjs.org) - Read more about Angular in the Frameworks of choice section further up on this page.
+ - [Ionic](https://www.ionicframework.com) - Read more about Ionic in the Frameworks of choice section further up on this page. 
+ - [Foundation for sites](https://foundation.zurb.com/sites/docs/) - Our favorite CSS framework and the perfect starter point for our client facing products, the modular structure and clean starting point makes it easy for us to go from prototypes to polished products.   
+ - [Foundation for emails](https://foundation.zurb.com/emails/docs) - Building the proper markup for emails is a nightmare, but this frameworks takes out most of that hazzle by providing an easy to use templating language and a set of tested UI components.
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: hazzle -> hassle

File: source/our-stacks/front-end/index.md [hunk 8]
```diff
@@ -3,4 +3,80 @@ title: Front-end
...
+ - [ui.router](https://angular-ui.github.io/ui-router/site/) - The best routing solution for Angular.
+ 
+### Open source
+To support our daily work we maintain a range of open source libraries, ranging from Angular based UI components, to NodeJS tooling, that helps us ensure high quality products and give us battle-tested components that are easy to use. All our repositories are being maintained on [Github](https://github.com/nodes-frontend/).
+
+ - [generator-nodes](https://www.github.com/nodes-frontend/generator-nodes) - Our opinionated Yeoman CLI, and the boilerplate for all of our Angular projects.
+ - [nodes-component-starter](https://www.github.com/nodes-front-end/nodes-component-starter) - Not every project ends up being an actual product, we often build isolated UI components and small libraries to use in our projects. This is our starting point for Open Source projects and comes with Angular, Typescript, semantic-release and slew of other awesome features.
+ - [nImage](https://www.github.com/nodes-frontend/nImage) - We use images, alot. This module enhances images with (lazy-)loading, fallbacks, aspect-ratios and cropping/resizing.
```
Comment (chriscombs) [inline source/our-stacks/front-end/index.md]: alot -> a lot

**Pull request merged**